docs/sending.md

```diff
@@ -13,6 +14,88 @@ graph LR
     style S4 fill:#eef4fd,stroke:#4285F4
 ```
 
-### How can I cancel a transaction on Chainsfer?
-To be added later...
-Choose your wallet --> Choose the coin --> Enter transfer information --> Review and Confirm
\ No newline at end of file
+### 1. Choose your wallet
+
+The first step is to choose a wallet from which funds is sent out.
+We support the following three wallets:
+
+> * [Drive Wallet](/wallet)
+> * [Metamask](https://metamask.io/)
+> * [Ledger](https://www.ledger.com/)
+
+Click on the wallet you want to use and proceed to the next step. 
+If you are using a hardware wallet, make sure to have your device plugged.
+
+### 2. Choose the cryptocurrency
+
+Click on the type of cryptocurrency you want to send in the coin list, and following the instruction on the page.
+
+### 3. Enter recipient information
+
+We autofill the sender email. You only need to provide the following:
+
+> * Recipient Email
+> * Amount
+> * Security Answer (self-provided or auto-generated)
+
+### 4. Review and confirm
+
+Please double check the details with your recipient, such as recipient email address, amount and security answer, and make sure they're correct before confirming your transfer. The transaction fee is also estimated and shown on the review page.
+
+Click the "Confirm and Transfer" button to continue, and following the instruction at the bottom of the page.
+
+Once it is completed, a receipt is shown. Be sure to copy the security answer and send it to your recipient.
+
+## How long does a transfer take?
+
+A single transfer involves two cryptocurrency transactions, one for sending and one for receiving.
+
+The two main factors influencing the transaction time are:
+    1. The amount of network activity
+    2. Transaction fees
+
+We choose the median transaction fee from the last several blocks, assuming a typical daily transaction volume,
```

Comment:
Chainsfer automatically chooses the **median of transaction fees** from the last several blocks